src/SimpleAzure.Storage.HybridQueue/HybridQueue.cs

```diff
@@ -70,7 +57,7 @@ public async Task AddMessageAsync<T>(T item, TimeSpan? initialVisibilityDelay, C
 
             var blobId = Guid.NewGuid().ToString(); // Unique Name/Identifier of this blob item.
             var binaryData = new BinaryData(message);
-            await _blobContainerClient.UploadBlobAsync(blobId, binaryData, cancellationToken);
+            await _blobContainerClient.UploadBlobAsync(blobId, binaryData, cancellationToken).ConfigureAwait(false);

```

Comment:
Why CA(f)? I was under the impression that we can finally get rid of that for most* scenario's, in .NET 'core' (and newer).

I'm not really wanting to target FF.

Reply:
See my reply to Phillip, above. Interesting that you're both under the impression that something about this changed in .NET Core. Have a source on that? I've never heard it

Reply:
As replied earlier - Approved and crying and 🤮